### PR TITLE
[KRK-2024] Krakow 2024 - Initial PR

### PR DESCRIPTION
**Pull request by ivellios**
**This is a draft** - missing links to the proper CFP and registration pages as these are blocked by the Core Team accept for the event and update of the mailing list/group.

- Initial information (date/location) added for Kraków 2024 event
- Updated list of organizers along with photos

### CODE REVIEW

File: data/events/2024-krakow.yml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+description: "5th edition of DevOpsDays in Kraków!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted 2021-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2021-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: 2023-03-12T00:00:00+02:00
+enddate: 2023-03-13T23:59:59+02:00
```
Comment (adrianmoisey) [inline data/events/2024-krakow.yml:16]:
```suggestion
startdate: 2024-03-12T00:00:00+02:00
enddate: 2024-03-13T23:59:59+02:00
```

Comment (adrianmoisey) [reply data/events/2024-krakow.yml:16]: The page doesn't get added to the front page of DOD at the moment, as these dates are in the past.

**Comment by adrianmoisey** [comment]
I added `awaiting-email-to-info` as there is an email thread about getting the correct URL for the CFP.

File: static/events/2024-krakow/organizers/jakub_barc.png
Comment (adrianmoisey) [inline static/events/2024-krakow/organizers/jakub_barc.png:1]: Could you convert this file to a smaller JPEG?

Comment (ivellios) [reply static/events/2024-krakow/organizers/jakub_barc.png:1]: Sure can do!

**Review by jyee — CHANGES_REQUESTED** [review]
One additional minor change: Please update the `/static/_redirects` file (line 63) to redirect `/krakow` to this event. It currently directs to the 2020 event.

In the future, we recommend using the `add_new_event.sh` script in the [Utilities](https://github.com/devopsdays/devopsdays-web/tree/main/utilities) directory to generate your events (that will ensure they have the proper site updates).

Also, per the email exchange, I think that the `2024-krakow.yml` file needs the CFP info, but this is not a blocker if you intend to add that later for some reason.

Aside from this, everything else appears to be ok, so mark the PR as "reviewable" (it is currently "draft") and someone from the core team can merge it.

**Comment by ivellios** [comment]
@jyee - I have addressed your notes. Thank you for the suggestions.

We have also pushed the date of the event and had to change the location.
I mark it as ready for review.

**Pull request merged**